web/packages/teleterm/electron-builder-config.js

```diff
@@ -67,6 +102,7 @@ module.exports = {
     ].filter(Boolean),
   },
   dmg: {
+    artifactName: '${productName}-${version}-${arch}.${ext}',
```

Comment:
I know that some customers have scripts set up which automatically download Connect when setting up new machines.

Won't this change break those scripts? Do you know if we'll have to update the Downloads page as well?